Sources/SignHereLibrary/Models/Certificates/CertificateType.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  CertificateType.swift
+//  Models
+//
+//  Created by Caleb Davis on 06/21/23.
+//
+
+import ArgumentParser
+import Foundation
+
+internal enum CertificateType: String, Codable, ExpressibleByArgument {
```

Comment:
It might be nice to contextualize these values by including a comment containing a link to the definition (as in the command parameter): https://developer.apple.com/documentation/appstoreconnectapi/certificatetype